modules/nodes-nodes-resources-configuring-about.adoc

```diff
@@ -29,65 +28,35 @@ introduction of allocatable resources.
 
 [id="allocate-node-enforcement_{context}"]
 == How nodes enforce resource constraints
 
-The node is able to limit the total amount of resources that pods
-may consume based on the configured allocatable value.  This feature significantly
-improves the reliability of the node by preventing pods from starving
-system services (for example: container runtime, node agent, etc.) for resources.
-It is strongly encouraged that administrators reserve
-resources based on the desired node utilization target
-in order to improve node reliability.
-
-The node enforces resource constraints using a new *cgroup* hierarchy
-that enforces quality of service.  All pods are launched in a
-dedicated cgroup hierarchy separate from system daemons.
-
-Optionally, the node can be made to enforce kube-reserved and system-reserved by
-specifying those tokens in the enforce-node-allocatable flag.  If specified, the
-corresponding `--kube-reserved-cgroup` or `--system-reserved-cgroup` needs to be provided.
-In future releases, the node and container runtime will be packaged in a common cgroup
-separate from `system.slice`.  Until that time, we do not recommend users
-change the default value of enforce-node-allocatable flag.
-
-Administrators should treat system daemons similar to Guaranteed pods.  System daemons
-can burst within their bounding control groups and this behavior needs to be managed
-as part of cluster deployments.  Enforcing system-reserved limits
-can lead to critical system services being CPU starved or OOM killed on the node. The
-recommendation is to enforce system-reserved only if operators have profiled their nodes
-exhaustively to determine precise estimates and are confident in their ability to
-recover if any process in that group is OOM killed.
-
-As a result, we strongly recommended that users only enforce node allocatable for
-`pods` by default, and set aside appropriate reservations for system daemons to maintain
-overall node reliability.
+The node is able to limit the total amount of resources that pods can consume based on the configured allocatable value. This feature significantly improves the reliability of the node by preventing pods from using CPU and memory resources that are needed by system services such as the container runtime and node agent. To improve node reliability, administrators should reserve resources based on a target for resource use.
+
+The node enforces resource constraints using a new cgroup hierarchy that enforces quality of service. All pods are launched in a dedicated cgroup hierarchy that is separate from system daemons.
+
+Administrators should treat system daemons similar to pods that have a guaranteed quality of service. System daemons can burst within their bounding control groups and this behavior must be managed as part of cluster deployments. Reserve CPU and memory resources for system daemons by specifying the amount of CPU and memory resources to reserve in `system-reserved`.
```

Comment:
Do we want _reserve_ used twice in this sentence?

Reply:
Removed. Thanks for pointing it out.